### PR TITLE
add 自主研究事業2020

### PR DESCRIPTION
**Pull request by abebe9849**
確認お願いします。

### CODE REVIEW

**Review by atamaokac — COMMENTED** [review]
I corrected some trivial errors. I recommend someone else to review it.
(For me the photo looks a bit too large...)

File: content/articles/2021sy/blog/jisyukenkyu.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Title: 自主研究奨励事業2020
+Date: 2021.05.06
+Modified: 2021.05.06
+Tags: Machine Learning
+Authors: 安部,梅田,富田
+
+学部学生による自主研究奨励事業、全学選抜自主研究成果発表会
```
Comment (takyamamoto) [inline content/articles/2021sy/blog/jisyukenkyu.md:7]: You should add 「令和2年度」学部学生による自主研究奨励事業

File: content/articles/2021sy/blog/jisyukenkyu.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+2021/04/30に銀杏会館でとりおこなわれた発表会にて優秀賞をいただきました。
+![写真]({attach}./images/jisyukenkyu_figs/8519.jpg)
```
Comment (takyamamoto) [inline content/articles/2021sy/blog/jisyukenkyu.md:10]: The image size is large, so you should use 
`jisyukenkyu_figs8519`

File: content/articles/2021sy/blog/jisyukenkyu.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+テーマは「眼底画像から緑内障か否か判断する深層学習モデル開発」でした。
+アドバイザー教員の三木 篤也先生をはじめ、様々な方にご支援いただきましたことを心から感謝申し上げます。
+計算機として、[医学科学生用計算機]({filename}/pages/student_server.md)を活用しました。
+
+詳細は[こちら](https://ir.library.osaka-u.ac.jp/repo/ouka/all/search/200220010496/?lang=0&cate_schema=3000&mode=0&cflg=1&codeno=journal)となります。
```
Comment (takyamamoto) [inline content/articles/2021sy/blog/jisyukenkyu.md:15]: The link should be `http://hdl.handle.net/11094/80643`.

Comment (takyamamoto) [reply content/articles/2021sy/blog/jisyukenkyu.md:15]: Can you use this image in your paper?
image

**Review by takyamamoto — CHANGES_REQUESTED** [review]

File: content/articles/2021sy/blog/jisyukenkyu.md (hunk 3 above)
Comment (abebe9849) [reply content/articles/2021sy/blog/jisyukenkyu.md:15]: OK.

**Review by takyamamoto — APPROVED** [review]

**Pull request merged**